challenges/2-tabuada/php/lasatd/tabuada.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$num = $argv[1];
+
+    for($i = 1; $i <= $num; $i++ ){
```

Comment:
Não há necessidade de indentação aqui. Você só deve indentar a parte interna de um bloco de código como um `if`, `for`, `while` etc.